query-service-impl/src/main/java/org/hypertrace/core/query/service/prometheus/PrometheusBasedResponseBuilder.java

```diff
@@ -0,0 +1,235 @@
+package org.hypertrace.core.query.service.prometheus;
+
+import java.time.Instant;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+import okhttp3.Request;
+import org.hypertrace.core.query.service.api.Row;
+import org.hypertrace.core.query.service.api.Row.Builder;
+import org.hypertrace.core.query.service.api.Value;
+import org.hypertrace.core.query.service.prometheus.PromQLMetricResponse.PromQLMetricResult;
+
+public class PrometheusBasedResponseBuilder {
+
+  /*
+   * @param promQLMetricResponseMap map of prometheus query and its response
+   * @param columnNameToAttributeMap map of attribute id to prometheus metric attribute name
+   * @param columnNameToQueryMap map of attribute id of metric to corresponding promQL query
+   * @param columnSet set of all attribute ids of returned column meta data set
+   * @param timeStampColumn attribute id of time stamp column
+   *
+   * E.g
+   * columnNameToAttributeMap :  (SERVICE.id, service_id)
+   * columnNameToQueryMap (SERVICE.numCalls, sum by (service_id) (sum_over_time(num_calls{}[]))
+   * columnSet : [SERVICE.startTime, SERVICE.id, SERVICE.numCall]
+   * timeStampColumn : SERVICE.startTime
+   * */
+  static List<Row> buildResponse(
+      Map<Request, PromQLMetricResponse> promQLMetricResponseMap,
+      Map<String, String> columnNameToAttributeMap,
```

Comment:
nit: can we rename the param names?
```
Map<String, String> logicalAttributeNameToMetricAttributeMap,
Map<String, String> logicalAttributeNameToMetricQueryMap,
List<String> columnSelectionSet,
```

Reply:
done. All remove some of the unused code.